runners/flink/src/test/validatesRunnerConfig/flink-conf.yaml

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+#parallelism.default: 23
+taskmanager.memory.network.fraction: 0.2
+taskmanager.memory.network.max: 1gb
+pipeline.operator-chaining.enabled: false
```

Comment:
Is this OK? I guess it forces more serialization so everything will get slower during testing, but coder errors will be caught. Could it also mask errors?

Reply:
Hum tbh I don't really remember why I changed that. I'll check maybe this is not required.